Jantimizer/QueryParser/QueryParsers/JoinQueryParser.cs

```diff
@@ -80,9 +83,24 @@ private JoinNode ParseIntoModel(string subQuery)
                     rightTable = onSplit[0].Trim();
                 if (onSplit.Length > 1)
                     condition = onSplit[1].Trim();
+                if (condition.Length > 1)
+                {
+                    string[] conditionSplit = {"", ""};
+                    foreach (var op in Utilities.Operators) {
+                        if (condition.Contains(op.Item2)) {
+                            conditionSplit = condition.Split(op.Item2);
+                            type = op.Item1;
+                            break;
+                        }
+                    }
+                    if (conditionSplit[0].Contains('.'))
+                        leftAttribute = conditionSplit[0].Split('.')[1].Trim();
+                    if (conditionSplit[1].Contains('.'))
+                        rightAttribute = conditionSplit[1].Split('.')[1].Trim();
```

Comment:
This still wouldnt work with the `AND` and `OR` operators, right?
Again just make it an Issue if its not easy to fix.